helmfile/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -4,3 +4,35 @@ config:
   modules:
     tcp_connect:
       prober: tcp
+
+serviceMonitor:
+  enabled: true
+  targets:
+    {{- if .Values.prometheusBlackboxExporter.targets.rook }}
+    - name: rook
+      url: rook-ceph-mgr.rook-ceph.svc.cluster.local:9283
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
+    {{- end }}
```

Comment:
I am not sure if the probe method was suggested in wc cluster as per ADR - https://github.com/elastisys/compliantkubernetes/pull/241 . Not sure if this applies in this case. @cristiklein  would be best person to answer this .